src/Transition/InputRulesFiles/Rules9-5-0-to-9-6-0.md

```diff
@@ -30,3 +30,12 @@ Field 1-3 remain the same.
 Field 4 has been eliminated/deleted and not replace with anything else.
 
 See [8884](https://github.com/NREL/EnergyPlus/pull/8884)
+
+# Object Change: PerformancePrecisionTradeoffs
+Summary: A new override option including cubic spline interpolations in replacement of original psychrometric function PsyTsatFnPb was added as Mode06 in this object. The previous options for the override mode of the object including Mode06 and Mode07 in version 9.5 were switched to Mode07 and Mode08 in version 9.6, respectively.
+
+Fields 1-2 remains the same.
+Fields 3 has been updated by adding a new override mode option. 
```

Comment:
Could be described a bit better in this line, but I guess the paragraph above basically says it.